timetable/semester/weekend/Shuttlecock_I_weekend.json

```diff
@@ -1,58 +1,18 @@
 {
   "shuttlecock_i_weekend": [
```

Comment:
This schedule is for Shuttlecock inbound. All the types should be "R", meaning Residence.